src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -111,11 +110,6 @@ public MicrobeCamera Camera
         }
     }
 
-    /// <summary>
-    ///   The selected membrane rigidity
-    /// </summary>
-    public float Rigidity { get; private set; }
```

Comment:
I think that this variable should be kept as (or at least the original rigidity is needed)
as the rigidity changing needs to take something like 30 MP (but only the first time the slider is moved in a editor session) it also needs to add an undo action so that the rigidity can be reset (back to the value on entry to the editor) to get back the ATP.